dev/breeze/src/airflow_breeze/utils/packages.py

```diff
@@ -177,7 +177,7 @@ def validate_provider_info_with_runtime_schema(provider_info: dict[str, Any]) ->
         jsonschema.validate(provider_info, schema=schema)
     except jsonschema.ValidationError as ex:
         get_console().print("[red]Provider info not validated against runtime schema[/]")
-        raise Exception(
+        raise ValueError(
```

Comment:
sys.exit() here with epxlaination which `provider.yaml` file should be fixed + error.message to explain the validation error.